app/assets/javascripts/experiments/table.js

```diff
@@ -21,14 +21,26 @@ var ExperimnetTable = {
   },
   appendRows: function(result) {
     $.each(result, (id, data) => {
-      // Checkbox selector
-      let row = `
-        <div class="table-body-cell">
-          <div class="sci-checkbox-container">
-            <input type="checkbox" class="sci-checkbox my-module-selector" data-my-module="${id}">
-            <span class="sci-checkbox-label"></span>
-          </div>
-        </div>`;
+      let row;
+      let provisioningStatus = data.provisioning_status;
+
+      // Checkbox selector or provisioning spinner
+      if (provisioningStatus === 'in_progress') {
```

Comment:
Can we just use class to row during provisioning to hide/show required information?